### ISSUE TITLE
Not compiling on RHEL 6.7

### ISSUE THREAD

**Issue by fnollet** [mummer4/mummer#19]
Installed good version for automake/autoconf, also make yaggo available to make sure I can compile the develop branch, but no luck:

[bbnof@gquest] mummer $ autoreconf --version
autoreconf (GNU Autoconf) 2.69
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<http://gnu.org/licenses/gpl.html>, <http://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.

[bbnof@gquest] mummer $ automake --version
automake (GNU automake) 1.14.1
Copyright (C) 2013 Free Software Foundation, Inc.
License GPLv2+: GNU GPL version 2 or later <http://gnu.org/licenses/gpl-2.0.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by Tom Tromey <tromey@redhat.com>
       and Alexandre Duret-Lutz <adl@gnu.org>.

[bbnof@gquest] mummer $ which yaggo
/tools/general/app/yaggo-1.5.9/bin/yaggo

[bbnof@gquest] mummer $ autoreconf -fi
configure.ac:8: error: possibly undefined macro: AC_PROG_LIBTOOL
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
autoreconf: /tools/general/app/autoconf-2.69/bin/autoconf failed with exit status: 1

Any tips or help here? Thanks!

**Comment by fnollet** [comment]
Also plan on using GCC 4.9.4...

**Comment by gmarcais** [comment]
You need to have [libtool](https://www.gnu.org/software/libtool/) installed.

**Comment by fnollet** [comment]
This is the case: 

[root@gquest] modulefiles # rpm -q libtool
libtool-2.2.6-15.5.el6.x86_64

**Comment by fnollet** [comment]
Any update here? I am stuck at this moment...

**Comment by fnollet** [comment]
Hi

I guess installing separate autoconf/automake packages, caused libtool to malfunction?
I had to explicitly set: 
export ACLOCAL_PATH=/usr/share/aclocal

Using the libtool from redhat caused lots of warning however, so I compiled/installed libtool 2.4.6 and set the above env var to include both system aclocal and dedicated installation aclocal. 

This caused the error to go away. 

Thanks for your support. 
Filip
